index.html

```diff
@@ -1102,6 +1130,17 @@ <h2 id="about">About</h2>
       <p>Made with 💛 by <a href="https://vlaservi.ch" target="_blank">Rhyne</a> and some <a href="https://github.com/rhyneav/papercss/graphs/contributors">fantastic contributors</a>!</p>
     </div>
   </div>
+  <script>
+    /* Toggle between adding and removing the "responsive" class to topnav when the user clicks on the icon */
+    function myFunction() {
+      var x = document.getElementById("myTopnav");
+      if (x.className === "topnav") {
+          x.className += " responsive";
+      } else {
+          x.className = "topnav";
```

Comment:
This wont be nice if the nav bar uses other classes

Reply:
I will rewrite it! Thank you for feedback!